firstvoices/backend/serializers/widget_serializers.py

```diff
@@ -36,3 +36,30 @@ class Meta(SiteContentLinkedTitleSerializer.Meta):
             "format",
             "settings",
         )
+
+
+class SiteWidgetListOrderDetailSerializer(SiteWidgetDetailSerializer):
+    order = serializers.SerializerMethodField()
+
+    class Meta(SiteWidgetDetailSerializer.Meta):
+        model = SiteWidget
+        fields = (
+            "id",
+            "order",
+            "title",
+            "widget_type",
+            "format",
+            "visibility",
+            "settings",
+        )
+
+    def get_order(self, widget):
+        return widget.sitewidgetlistorder_set.all().first().order
+
+
+class SiteWidgetListSerializer(serializers.ModelSerializer):
+    widgets = SiteWidgetListOrderDetailSerializer(many=True)
```

Comment:
I'd be curious about what @gmcauliffe thinks, but I suspect it would be fine to serialize the homepage/page widgets as an ordered list of widget objects, and not need to include the actual list order number.

Reply:
That would be fine for the FE 👍